.pre-commit-config.yaml

```diff
@@ -42,8 +42,7 @@ repos:
     rev: v1.11.0
```

Comment:
```suggestion
    rev: v1.17.0
```

This is the version we're using on `main`, and it ensures that the `3.10` classifier sorts in version order (i.e. after `3.9`).

Reply:
Done, thanks!